crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -1027,6 +1033,93 @@ impl<F: Clone + Send> Domain<F, api::PaymentsRequest, PaymentData<F>> for Paymen
         Ok(())
     }
 
+    #[allow(clippy::too_many_arguments)]
+    async fn call_unified_authentication_service_if_eligible<'a>(
+        &'a self,
+        state: &SessionState,
+        payment_data: &mut PaymentData<F>,
+        _should_continue_confirm_transaction: &mut bool,
+        _connector_call_type: &ConnectorCallType,
+        business_profile: &domain::Profile,
+        key_store: &domain::MerchantKeyStore,
+    ) -> CustomResult<(), errors::ApiErrorResponse> {
+        let is_click_to_pay_enabled = true;
```

Comment:
Avoid hard coding the data in code, get from env or profiles

Reply:
we will fetch the data from profiles in upcoming pr's for now changing it to todo!